docker/develop/develop.dockerfile

```diff
@@ -7,9 +7,6 @@ FROM registry.opensuse.org/opensuse/leap:15.3
 ENV container docker
 ENV DISTRO SUSE
 
-# Update Leap to most current packages
-RUN zypper update -y
-
```

Comment:
Why are we removing this?

Reply:
Because containers are rebuilt on OBS side and thus don't need it.